invenio_communities/communities/services/customizations.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
```

Comment:
same here i.e copied from invenio-rdm-records.

Reply:
Opened this [issue](https://github.com/inveniosoftware/invenio-records-resources/issues/382) to move the code